### ISSUE TITLE
Memory leak

### ISSUE THREAD

**Issue by ignl** [collab-project/videojs-wavesurfer#128]
## Description
During navigation I destroy wavesurfer player with microphone plugin, but it seems memory is not exactly freed and slowly increasing by creating/destro
![1](https://user-images.githubusercontent.com/2720602/109808874-4acaa100-7c30-11eb-9b23-52e6626d4500.png)
![2](https://user-images.githubusercontent.com/2720602/109808878-4c946480-7c30-11eb-87ea-6fee0952a73c.png)

ying player.

## Versions
Make sure to include the following versions:
### videojs/wavesurfer
Video.js: 7.11.4
Video.js wavesurver: 3.3.1
wavesurfer.js: 4.4.0
videojs-record: 4.2.0

**Comment by thijstriemstra** [comment]
thanks for the report. A memory issue was resolved in wavesurfer.js 4.5.0, can you give that a try @ignl?

**Comment by ignl** [comment]
Yes, it's the same thing with new version as well. It seems that videojs wavesurfer plugin still holds reference to videojs player through some handlers(?) that are not cleared. Not sure haven't checked the code myself yet, maybe you might know right away.

**Comment by thijstriemstra** [comment]
can you attach a small example that reproduces the issue?

**Comment by ignl** [comment]
I have tried first to modify regular example videojs-wavesurfer plugin audio example and it seems it works ok. However in my example I used wavesurfer plugin with audio recorder and after adding that into example memory leak is visible clearly when pressing init/destroy buttons and doing allocation instrumentation (Memory tab in google chrome dev tools).
I use google chrome 88 on kubuntu 20.10. It seems some handlers are not cleared, but since it's related to recorder maybe the problem is there...
[example.zip](https://github.com/collab-project/videojs-wavesurfer/files/6077733/example.zip)
